cmd/collectors/rest/plugins/snapmirror/snapmirror.go

```diff
@@ -9,18 +9,22 @@ import (
 	"github.com/netapp/harvest/v2/cmd/tools/rest"
 	"github.com/netapp/harvest/v2/pkg/conf"
 	"github.com/netapp/harvest/v2/pkg/matrix"
+	"github.com/netapp/harvest/v2/pkg/tree/node"
 	"github.com/tidwall/gjson"
+	"strings"
 	"time"
 )
 
 const PluginInvocationRate = 10
 
 type SnapMirror struct {
 	*plugin.AbstractPlugin
+	data           *matrix.Matrix
 	client         *rest.Client
 	query          string
 	nodeUpdCounter int
 	svmVolToNode   map[string]string
+	svmPeerDataMap map[string]string // [peer SVM alias name] -> [peer SVM actual name] - [peer cluster name] map
```

Comment:
would it be clearer to create a local struct for the peer and then use something like `svmPeerDataMap map[string]peer`

Reply:
added